spec/lib/lorekeeper/json_logger_spec.rb

```diff
@@ -21,13 +21,13 @@
           logger.send(method, message)
           expect(io.received_message).to eq(expected)
         end
-        it 'The first key is message' do
+        it 'The first key is timestamp' do
```

Comment:
mind changing these so that it reads properly with the `it`, something like `it 'has the timestamp in the first key' do`